feedTheMonster.ts

```diff
@@ -87,8 +87,9 @@ class App {
     }
     this.registerWorkbox();
 
-    this.loadTitleFeedbackCustomFont();
-    this.titleTextElement.classList.add("animate");
+    // load the title custom font before showing it
+    await this.loadTitleFeedbackCustomFont();
+    this.titleTextElement.classList.add("show");
```

Comment:
this is not required as by default class can be applied and remove when required

Reply:
+1